src/cli.rs

```diff
@@ -41,19 +43,51 @@ pub fn parse(target_list: &TargetList) -> Args {
                     all.push(format!("--target-dir=/target"));
                 }
             } else {
-              if !arg.starts_with('-') && sc.is_none() {
-                  sc = Some(Subcommand::from(arg.as_ref()));
-              }
+                if !arg.starts_with('-') && sc.is_none() {
+                    sc = Some(Subcommand::from(arg.as_ref()));
+                }
 
-              all.push(arg.to_string());
+                all.push(arg.to_string());
             }
         }
     }
 
+    let docker_in_docker = parse_bool_env(env::var("CROSS_DOCKER_IN_DOCKER"));
+
     Args {
         all,
         subcommand: sc,
         target,
         target_dir,
+        docker_in_docker,
+    }
+}
+
+fn parse_bool_env(val: Result<String, env::VarError>) -> bool {
```

Comment:
I don't think this needs to be a function, given that it is only used once.

Reply:
True. Given that there is no dedicated CI test for docker-in-docker, I at least wanted to test that the environment variable is parsed correctly.

Reply:
@reitermarkus Would you prefer me to remove the test again, or keep it as is?

Reply:
Let's remove it. It's mainly testing `FromStr for bool`, which should be tested plenty already.

Reply:
Done.